docker-compose.yml

```diff
@@ -1,4 +1,4 @@
-version: "3.8"
+version: "2.4"
```

Comment:
For me the docker compose version wasn't an issue and the service got the IPv6 correctly assigned.
Which version of the docker engine did you use?
Mine is

> Docker version 20.10.12, build e91ed57

Reply:
Interesting.  It seems I'm also on the same version.
```
Server: Docker Engine - Community
 Engine:
  Version:          20.10.12
  API version:      1.41 (minimum version 1.12)
  Go version:       go1.16.12
  Git commit:       459d0df
  Built:            Mon Dec 13 11:43:56 2021
  OS/Arch:          linux/amd64
  Experimental:     false
```
Not sure what to make of it.  Testing on a Debian 11 box.

Reply:
Ok I just read through your links to the docker compose issue again and I don't know why it works on my VPS and laptop.
I think this should stay on version 2.4 for now.